src/Core__Date.resi

```diff
@@ -0,0 +1,294 @@
+@send external setHoursMS: (t, ~hours: int, ~minutes: int, ~seconds: int) => unit = "setHours"
+@send
+external setHoursMSMs: (t, ~hours: int, ~minutes: int, ~seconds: int, ~milliseconds: int) => unit =
+  "setHours"
+@send external setMinutes: (t, int) => unit = "setMinutes"
+@send external setMinutesS: (t, ~minutes: int, ~seconds: int) => unit = "setMinutes"
+@send
+external setMinutesSMs: (t, ~minutes: int, ~seconds: int, ~milliseconds: int) => unit = "setMinutes"
+@send external setSeconds: (t, int) => unit = "setSeconds"
+@send external setSecondsMs: (t, ~seconds: int, ~milliseconds: int) => unit = "setSeconds"
+@send external setMilliseconds: (t, int) => unit = "setMilliseconds"
+@send external setDay: (t, int) => unit = "setDay"
+@send external getUTCFullYear: t => int = "getUTCFullYear"
+@send external getUTCMonth: t => int = "getUTCMonth"
+@send external getUTCDate: t => int = "getUTCDate"
+@send external getUTCHours: t => int = "getUTCHours"
+@send external getUTCMinutes: t => int = "getUTCMinutes"
+@send external getUTCSeconds: t => int = "getUTCSeconds"
+@send external getUTCMilliseconds: t => int = "getUTCMilliseconds"
+@send external getUTCDay: t => int = "getUTCDay"
+@send external setUTCFullYear: (t, int) => unit = "setUTCFullYear"
+@send external setUTCFullYearM: (t, ~year: int, ~month: int) => unit = "setUTCFullYear"
+@send
+external setUTCFullYearMD: (t, ~year: int, ~month: int, ~date: int) => unit = "setUTCFullYear"
+@send external setUTCMonth: (t, int) => unit = "setUTCMonth"
+@send external setUTCDate: (t, int) => unit = "setUTCDate"
+@send external setUTCHours: (t, int) => unit = "setUTCHours"
+@send external setUTCHoursM: (t, ~hours: int, ~minutes: int) => unit = "setUTCHours"
+@send
+external setUTCHoursMS: (t, ~hours: int, ~minutes: int, ~seconds: int) => unit = "setUTCHours"
+@send
+external setUTCHoursMSMs: (
+  t,
+  ~hours: int,
+  ~minutes: int,
+  ~seconds: int,
+  ~milliseconds: int,
+) => unit = "setUTCHours"
+@send external setUTCMinutes: (t, int) => unit = "setUTCMinutes"
+@send external setUTCMinutesS: (t, ~minutes: int, ~seconds: int) => unit = "setUTCMinutes"
+@send
+external setUTCMinutesSMs: (t, ~minutes: int, ~seconds: int, ~milliseconds: int) => unit =
+  "setUTCMinutes"
+@send external setUTCSeconds: (t, int) => unit = "setUTCSeconds"
+@send external setUTCSecondsMs: (t, ~seconds: int, ~milliseconds: int) => unit = "setUTCSeconds"
+@send external setUTCMilliseconds: (t, int) => unit = "setUTCMilliseconds"
+@send external setUTCDay: (t, int) => unit = "setUTCDay"
+@send external toDateString: t => string = "toDateString"
+@send external toString: t => string = "toString"
+@send external toTimeString: t => string = "toTimeString"
+
+/** 
+  `toLocaleDateString(date)`
+
+  Converts a JavaScript date to a localized date string. It will use the current locale.
+
+  ## Examples
+  ```rescript
+  Date.make()->Date.toLocaleDateString->Console.log
```

Comment:
Could you also add a comment with what the output will be when you log to the console here? Or rather, what the output is when you do it now as you're adding the docstrings.

This goes for all the examples.